cloud

```diff
@@ -1170,6 +1170,7 @@ function pubsub_subscription_withdraw() {
 #       --updated-topic=NAME
 #       --updated-debug-subscription=NAME
 #       --pick-notifications-trigger-topic=NAME
+#       --cache-topic=NAME
```

Comment:
🤔 why is this different?

Reply:
Ahh forgot to update the comments, thank you for pointing let me update it :)